README.md

```diff
@@ -2,6 +2,8 @@
 
 Welcome to the Postman Collections Quickstart Guide! If you're looking for a quick and easy way to get started with the Plaid API with no additional code, this is the place for you.
 
+To skip the instructions, you can go directly to [the Plaid Postman collection](https://www.postman.com/plaid-api/workspace/plaid/collection/12160321-04d27ed8-aed4-4d46-a1f8-70a72ec9a78a).
```

Comment:
Is this URL going to change every time you update the collection? Are you going to regret adding this?

Reply:
nope, this is a persistent url

Reply:
[]()

Reply:
[]()